src/bundle/Resources/public/js/scripts/admin.language.list.js

```diff
@@ -0,0 +1,10 @@
+(function () {
+    const checkboxes = [...document.querySelectorAll('form[name="languages_delete"] input[type="checkbox"]')];
```

Comment:
The button toggler should be used here. Requires: https://github.com/ezsystems/ezplatform-admin-ui/pull/145